app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= render "shared/header" %>
+  <div class="container">
+    <div class="add-btn">
+      <%= link_to 'recipes/new', :remote => true do %>
+      <input type="button" class="btn btn-primary mb-4" value="Add Recipe">
+      <% end %>
+    </div>
+
+      <% @recipes.each do |recipe| %>
+        <div class= "border border-1 shadow-sm p-3 d-flex justify-content-between mt-3">
+          <div>
+            <%= link_to recipe_path(recipe.id), :remote => true do %>
+            <h2><%=recipe.name%></h2>
+            <% end %>
+            <%= button_to "REMOVE", recipe_path(recipe.id), html: {'data-turbo' => "false"}, :method => :delete, class: "btn btn-danger" %>
```

Comment:
The user encounters this error while trying to delete a recipe `recipes index` page.

![remove_recipe](https://user-images.githubusercontent.com/35504149/175662233-d0433bbe-5ce5-4962-925d-212b238a4d17.png)

Reply:
- Added two `Foods`(before delete action)
![Before_Delete_Action](https://user-images.githubusercontent.com/83566118/175696858-6732d81d-9044-4ec8-bf3b-f2411c73418d.PNG)

- After Delete Action
![After_Delete_Action](https://user-images.githubusercontent.com/83566118/175696933-fa054f1f-d2da-408c-a30e-5dfb3a3b41b5.PNG)

Reply:
:+1: Fixed :heavy_check_mark: